pkg/docs/metrics_test.go

```diff
@@ -0,0 +1,36 @@
+package docs
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+)
+
+var _ = Describe("OperatorMetricsDoc", func() {
+	Context("RecordingRule Registration", func() {
+		It("should create metric and recording rule and check that they're documented", func() {
```

Comment:
if we are testing both metrics and recording rules here, which sounds good, the name of the context should be changed